lazy-either.js

```diff
@@ -1,46 +1,52 @@
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
-
-const LazyEither = require('lazy-either');
-const R = require('ramda');
-const S = require('sanctuary');
-
-
-//    join :: String -> String -> String
-const join = R.curryN(2, path.join);
+const fs            = require('fs');
+
+const LazyEither    = require('lazy-either');
+const S             = require('sanctuary');
+
+const exit0         = require('./common/exit0');
+const exit1         = require('./common/exit1');
+const join          = require('./common/join');
+
+
+//    Array$traverse :: Applicative f => TypeRep f -> (a -> f b) -> Array a -> f (Array b)
+//
+//    We cannot use S.traverse because LazyEither does not currently support
+//    recent versions of the Fantasy Land specification. It's thus necessary
+//    to dispatch to `map` and `ap` (rather than their prefixed equivalents).
```

Comment:
I'll remove the LazyEither solution, Risto. As I stated in Risto-Stevcev/lazy-either#2, Fluture and LazyEither are duplicative. There's little value in including both here.